### DISCUSSION TITLE
Register Meteor for OSS Fuzz

### DISCUSSION THREAD

**Discussion by jankapunkt** [meteor/meteor#12898] [Feature Requests]
https://google.github.io/oss-fuzz/

A worthy step to consider, once we are on Meteor 3.0 stable. What do you other think about this? Also, what does the core team think about this?

**Comment by StorytellerCZ** [comment]
Another thing that I found and partially filled out is this:
https://www.bestpractices.dev/en/projects/8085

I think that is pretty good thing for corporations. @fredmaiaarantes let me know if you want to update this as I have apparently become the owner of that particular entry. I will try to figure out how to add more people as managers.

**Reply by StorytellerCZ** [reply]
I have also tried to run https://github.com/mtov/truck-factor on Meteor, but we are either dead or there has been some error.

**Reply by fredmaiaarantes** [reply]
Hey @StorytellerCZ, could you see if you are able to add more people as managers?

**Reply by StorytellerCZ** [reply]
Doesn't look like it. I can either edit or delete. Nothing else. So we can adjust this together on a stream or I can delete and hence allow you to do it.
